staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -712,7 +712,7 @@ func expectConversionFailureMessage(id, message string) func(t *testing.T, ctc *
 			})
 		}
 		for _, subresource := range []string{"status", "scale"} {
-			for _, verb := range []string{"get", "udpate", "patch"} {
+			for _, verb := range []string{"get", "update", "patch"} {
 				t.Run(fmt.Sprintf("%s-%s", subresource, verb), func(t *testing.T) {
 					switch verb {
 					case "create":
```

Comment:
also, "create" is not in the list of verbs we loop over, so this is never called, and "get" is but has no corresponding case

Reply:
@liggitt which one do you think:
1. change `"get"` to `"create"`
2. add a branch for `"get"`

Reply:
@liggitt